server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -0,0 +1,80 @@
+package services.ti;
+
+import com.google.common.base.Strings;
+import forms.AddApplicantToTrustedIntermediaryGroupForm;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import javax.inject.Inject;
+import models.TrustedIntermediaryGroup;
+import play.data.Form;
+import repository.UserRepository;
+import services.DateConverter;
+
+public class TrustedIntermediaryService {
```

Comment:
suggestion (blocking): please document this class and make it `final`

Reply:
Done!